### PR TITLE
Portfolio Project Milestone: Contact form

### PR DESCRIPTION
**Pull request by mm-pazouki**
In this milestone, I:

Added contact section to page markup
Added contact form header
Added text to contact form
Added text input to the contact form
Added button to contact form
Styled contact form
Added background image to form section

### CODE REVIEW

**Review by Johnadibe — COMMENTED** [review]
Hi @mm-pazouki and @over-geek, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project that only has a single contribution, which involved pair programming. To ensure that everyone has contributed code to the project, please make sure that all team members have had the opportunity to participate. Additionally, since the website has not yet been hosted, kindly remove the live demo section.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: Index.html [hunk 1]
```diff
@@ -171,6 +171,79 @@ <h3 class="h3">Skills</h3>
             </ul>
         </section>
         
+        <section id="contact" class="contact">
+            <img src="./Contents/Resources/Pictures/image_geometry_1.svg" alt="" class="corner__geometry">
+            <img src="./Contents/Resources/Pictures/contact_shape_one.svg" alt="" class="contact__image__left">
+            <div class="article__form" >
+                <h2 class="h2 contact__h2">Get Started</h2>
+                <p class="contact__text">I Prefer To Speak By My Gestures</p>
+                <form action="https://formspree.io/f/" method="post" class="contact__form">
```
Comment (Strangeal) [inline Index.html]: - [x] Kindly consider adding a valid action link. In other, for you to do that please click on the [link](https://formspree.io/) provided in the project requirement and register an account. After registering an account you will be given a unique link which is what you're going to use in your form action to collect use data. Your unique link should look something like `https://formspree.io/f/{form_id}`. See below for more details: 
Screenshot 2023-05-02 at 19 02 06

File: Index.html [hunk 2]
```diff
@@ -171,6 +171,79 @@ <h3 class="h3">Skills</h3>
...
+                    <label class="label__name label" for="name">Your name <span class="red__colour">*</span></label>
+                    <input type="text" required maxlength="30" id="name" placeholder="write your name" name="name" class="name">
+                    <label class="label__email" for="email">Your email <span class="red__colour">*</span></label>
+                    <input type="email" required pattern="[a-z0-9._%+-]+@[a-z0-9.-]+\.[a-z]{2,}$" placeholder="write your email" id="email" name="email" class="email">
+                    <label class="label__message" for="message">Message <span class="red__colour">*</span></label>
+                    <textarea maxlength="500" required placeholder="Enter your message here..." name="message" id="message" class="message"></textarea>
+                    <span class="error" aria-live="polite"></span>
+                    <input type="submit" id="submit" value="Start Collaboration" class="buttons contact__button">
```
Comment (Strangeal) [inline Index.html:188]: - [x] Please make sure to stick to the `Figma` design template as much as possible. Your form `inputs` label on mobile screen doesn't show. Kindly fix it so your design looks exactly like that of the `Figma`design. See below for more details: 
<table>
<tr>
<th align="middle">Figma design</th>
<th align="middle">Your design</th>
</tr>
<tr>
<td>Screenshot 2023-05-02 at 18 57 21</td>
<td>Screenshot 2023-05-02 at 18 58 21</td>
</tr>
</table>

**Review by Strangeal — CHANGES_REQUESTED** [review]
Hi @mm-pazouki, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

Alt Text

## Highlights
Correct use of GitHub Flow ✅
Well-structured files ✅
All linter checks are passing ✅
Descriptive PR title with a great summary ✅

## Required Changes ♻️
- Congratulations 🎉 on the impressive work you've done so far. I've made some suggestions below to help you improve your project so kindly take a look at it and if you don't understand anything please don't hesitate to ask. Best of luck! 🤞🏻.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: Index.html (hunk 2 above)
Comment (Qoosim) [reply Index.html:188]: > * [x]  Please make sure to stick to the `Figma` design template as much as possible. Your form `inputs` label on mobile screen doesn't show. Kindly fix it so your design looks exactly like that of the `Figma`design. See below for more details:
> 
> Figma design	Your design
> Screenshot 2023-05-02 at 18 57 21	Screenshot 2023-05-02 at 18 58 21

- [ ] As suggested by the previous reviewer. This suggestion has not been attended to. Kindly do that.

### Your Design
your-design

### Figma Design
temp-design

**Review by Qoosim — CHANGES_REQUESTED** [review]
#  *Status: Required Changes* :recycle: 

Hi @mm-pazouki, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


## Highlights
-  No linter errors. :heavy_check_mark: 
-  Good README file. :heavy_check_mark: 
-  Correct GitHub flow. :heavy_check_mark: 



 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

- [x] Kindly fix the MIT License on your README file. It is broken at the moment.

license


_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by emmiiorji — APPROVED** [review]
Hi @mm-pazouki and Team, 

# Status: Approved ✅

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

<div align='center'>

</div>

## Good Highlights✔️

- Required changes have been impleneted.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**